contracts/src/v0.8/keystone/CapabilitiesRegistry.sol

```diff
@@ -467,7 +439,6 @@ contract CapabilitiesRegistry is OwnerIsCreator, TypeAndVersionInterface {
 
   /// @notice The next ID to assign a new DON to
   /// @dev Starting with 1 to avoid confusion with the zero value
-  /// @dev No getter for this as this is an implementation detail
   uint32 private s_nextDONId = 1;
 
   function typeAndVersion() external pure override returns (string memory) {
```

Comment:
Let's do something with the version here. Otherwise it is more difficult to track changes on deployed contracts.